spec/support/requests_shared_examples.rb

```diff
@@ -213,6 +213,32 @@
   end
 end
 
+shared_examples 'a sorted index resource' do |expected_last_item_sym|
+  let(:expected_last_item) { send(expected_last_item_sym) }
+  let(:sort_column) { :created_at }
+  let(:sort_order) { "desc" }
+
+  it 'should return a sorted index' do
+    is_expected.to eq(expected_response_status)
+    response_json = JSON.parse(response.body)
+    expect(response_json).to have_key('results')
+    returned_results = response_json['results']
+    expect(returned_results).to be_a(Array)
+    last_date = nil
+    returned_results.each do |this_result|
```

Comment:
I like what you're doing in this test with the looping over the results and testing the `sort_column`.